src/SFA.DAS.RoatpAssessor/SFA.DAS.RoatpAssessor.Web/Controllers/AssessorOutcomeController.cs

```diff
@@ -36,12 +36,19 @@ public async Task<IActionResult> AssessorOutcome(Guid applicationId)
 
             var viewModel = await _overviewOrchestrator.GetOverviewViewModel(new GetApplicationOverviewRequest(applicationId, userId));
 
-            if (viewModel is null || !viewModel.IsReadyForModeration)
+            if (viewModel is null || viewModel.IsAssessorApproved)
+            {
+                // This is in case the user presses the browser back button on AssessmentComplete
+                return RedirectToAction("Index", "Dashboard");
+            }
+            else if (!viewModel.IsReadyForModeration)
             {
                 return RedirectToAction("ViewApplication", "Overview", new { applicationId });
             }
-
-            return View("~/Views/AssessorOutcome/AssessorOutcome.cshtml", viewModel);
+            else
```

Comment:
this else is unnneded

Reply:
but it helps define the multiple flows which to me looks more readable 🤷‍♂️

Reply:
ok that's fine